client/src/pages/index.tsx

```diff
@@ -1,12 +1,180 @@
 import * as React from "react"
 import Layout from '../components/layout';
+// import {GridContainer, Grid} from '@trussworks/react-uswds';
+
+// this section seems verbose and silly
+// this inlines the svg as data:image/svg+xml ... this isnt' ideal.
```

Comment:
Can you explain this a bit? i'm not familiar with the tradeoffs!

Reply:
updated comment